src/foam/parse/QueryParser.java

```diff
@@ -133,6 +134,45 @@ private Grammar getGrammar() {
       return predicate;
     });
 
+    grammar.addSymbol("DOT", new Seq(grammar.sym("FIELD_NAME"), grammar.sym("SUB_QUERY")));
+    grammar.addAction("DOT", (val, x) -> {
+      Object[] values = (Object[]) val;
+      DotF dot = new DotF();
+      dot.setArg1(( values[1] instanceof Expr ) ?
```

Comment:
Would be nicer if DotF had a constructor that did this for you. Or a DOTF helper in mlang.java